packages/sanity/src/structure/comments/src/components/list/CommentsListItemLayout.tsx

```diff
@@ -325,7 +344,7 @@ export function CommentsListItemLayout(props: CommentsListItemLayoutProps) {
     >
       <InnerStack space={1} data-muted={displayError}>
         <Flex align="center" gap={FLEX_GAP} flex={1}>
```

Comment:
Good catch, updated it to use the avatar size prop. 
The height of the avatar will be variable, e.g. if size is 0 it will be 19px.
<img width="491" alt="Screenshot 2024-03-11 at 12 15 47" src="https://github.com/sanity-io/sanity/assets/46196328/10228141-5725-4f0f-8b80-aa20e67d840e">